bears/natural_language/WriteGoodLintBear.py

```diff
@@ -0,0 +1,68 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='write-good',
+        output_format='regex',
+        output_regex=r'(?P<message>.*)\s*on\s*line\s*(?P<line>\d+)\s*at\scolumn'
+                      '\s*(?P<column>\d+)'
+        )
+class WriteGoodLintBear:
+    """
+    Lints the text files using ``write-good`` for improving proses.
+
+    See <https://github.com/btford/write-good> for more information.
+    """
+    LANGUAGES = {"Natural Language"}
+    REQUIREMENTS = {NpmRequirement('write-good', '0.9.1')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Grammar'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         use_passive_voice: bool=False,
+                         check_so_beginning: bool=False,
+                         check_adverbs: bool=False,
+                         check_repeated_words: bool=False,
+                         check_is_beginning: bool=False,
+                         check_ambiguos_words: bool=False,
+                         check_extra_words: bool=False,
+                         check_cliche_phrases: bool=False):
+        """
+        Using ``False`` will disable the check.
```

Comment:
isn't it also disabled by False? shouldnt it be something like, using True will activate the check?